src/Symfony/Component/Routing/Loader/YamlFileLoader.php

```diff
@@ -78,6 +78,12 @@ public function load($file, $type = null)
         }
 
         foreach ($parsedConfig as $name => $config) {
+            if ('_subroutines' === $name && \is_array($config) && !isset($config['resource']) && !isset($config['path'])) {
```

Comment:
why check against for `resource` and `path` ?

Reply:
for BC

Reply:
Sorry, slipped my mind. :+1:

Reply:
Should we trigger a deprecation if somebody tries to use `_subroutines` as a route name here?